web/modules/custom/feedback/src/FeedbackInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Drupal\feedback;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\user\EntityOwnerInterface;
+use Drupal\Core\Entity\EntityChangedInterface;
+
+/**
+ * Provides an interface defining a Feedback entity.
+ *
+ * We have this interface so we can join the other interfaces it extends.
+ *
+ * @ingroup feedback
+ */
+interface FeedbackInterface extends ContentEntityInterface, EntityOwnerInterface, EntityChangedInterface {}
```

Comment:
what is the sense of this interface?
if no sense - don't create it

Reply:
I deleted this interface